cli/user/README.md

```diff
@@ -94,7 +95,52 @@ All available commands:
 	stream read scoped-stream-name [group-similar] [timeout-in-seconds]: Reads all Events from a Stream and then tails the Stream.
 ```
 
+You can try out commands such as:
+```
+> scope create example
+Scope 'example' created successfully.
+```
+
 For more info on how the Pravega User CLI works, please visit [this page](https://github.com/pravega/pravega/wiki/Pravega-User-CLI).
 
+## Pravega CLI on Kubernetes
+The pravega-cli can be accessed through the kubernetes pod such as the segmentstore pod.
+
+
+Once in the pod, you can run the pravega-cli under admin mode
+```
+    ./bin/pravega-cli
+    OpenJDK 64-Bit Server VM warning: Option MaxRAMFraction was deprecated in version 10.0 and will likely be removed in a future release.
+    Pravega User CLI Tool.
+        Usage instructions: https://github.com/pravega/pravega/wiki/Pravega-User-CLI
+    
+    Initial configuration:
+        controller-uri=localhost:9090
+        default-segment-count=4
+        timeout-millis=60000
+        max-list-items=1000
+        pretty-print=true
+```
+
+The initial configuration needs to be modified by using `config set` command.
```

Comment:
Please, make clear that there is only 1 parameter that needs to be set here, the `controller-uri`. The other parameters are related to the behavior of the CLI itself, but the user needs to understand that only 1 setting is necessary here.